Detectors/TRD/macros/calibrateKr.C

```diff
@@ -0,0 +1,38 @@
+#include "TRDCalibration/KrCalibration.h"
+
+using namespace o2::trd;
+
+void calibrateKr( TString sOpenFile )
```

Comment:
As we discussed, this macro does not run the actual calibration, but - as I understand it - reads the values for the calibrated pads from a tree, fills any gaps, subtracts the average, and fills the LocalGainFactor object. A name like CreateCCDBLocalGainFactor might be more appropriate.

Reply:
OK